src/app/game-state/battle.service.ts

```diff
@@ -325,27 +336,27 @@ export class BattleService {
       maxHealth: 20,
       accuracy: 0.5,
       attack: 5,
-      defense: 2,
+      defense: 5,
       loot: [
         this.itemRepoService.items['hide']
       ]
     },
     army: {
       name: "an angry army",
-      health: 2000000,
-      maxHealth: 20000000,
+      health: 20000000,
+      maxHealth: 20000000, 
       accuracy: 0.9,
-      attack: 1000000,
-      defense: 500000,
+      attack: 10000000,
+      defense: 10000000, // Army was a bit weak when I accidentally fought them, increased defense should help.
```

Comment:
Appreciate the FYI but don't think the comment needs to stay in the code.

Reply:
I mostly do that for the review so that when I'm coding at 2am you can see a comment from me when I forget to add comments in git from sleep deprivation and don't think about how this is permanently in code.